lib/knapsack_pro/repository_adapters/git_adapter.rb

```diff
@@ -14,8 +14,34 @@ def branches
         str_branches.split("\n")
       end
 
+      def authors
+        git_authors
+          .split("\n")
+          .map { |line| line.strip }
+          .map { |line| line.split("\t") }
+          .map do |commits, author|
+            { commits: commits.to_i, author: KnapsackPro::MaskString.call(author) }
+          end
+      rescue Exception
+        [{ commits: 0, author: "no git <no.git@example.com>" }]
+      end
+
+      def author
+        KnapsackPro::MaskString.call(git_author.strip)
```

Comment:
[A similar problem](https://github.com/KnapsackPro/knapsack_pro-ruby/pull/208/files#r1261027819) is for the `git_author` method. It could return an empty string when git command is not found in the system.

Maybe for the empty string (git_author) we should return `"no git <no.git@example.com>"`.

Reply:
See comment above